styles/globals.scss

```diff
@@ -0,0 +1,15 @@
+// import custom theme variables to override any bootstrap defaults
```

Comment:
can we move this file into "utils/theme"? the goal is that all files that need/can be updated per webstore instance be inside the "utils" folder. for the default layout anyway.

Reply:
remember to export it from "theme/index" if it's needed outside the "theme" folder.